### PR TITLE
Show file count and total size on top-level works; RDR-284.

### PR DESCRIPTION
**Pull request by coblej**

### CODE REVIEW

File: app/views/hyrax/datasets/_work_files.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<dl class="work-files">
```
Comment (mikedaul) [inline app/views/hyrax/datasets/_work_files.html.erb]: @coblej - what would you think about using some different markup here? I think it looks a little strange as a definition list and reads strangely with the word 'files' showing twice. Maybe just wrap it in a div with a unique class, change 'Files' to something like 'Size' and just use a separator (like an em-dash) to offset the count/size info? So something like: Size -- 2 files (300 Bytes)

File: app/services/work_files_scanner.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        else
+          scan_work(member_doc)
+        end
+      end
+    end
+  end
+
+  def scan_fileset(fileset_doc)
+    results.file_count = results.file_count + 1
+    results.file_size_total = results.file_size_total + fileset_doc['file_size_lts']
```
Comment (seanaery) [inline app/services/work_files_scanner.rb]: I'm getting a `nil can't be coerced into Fixnum` error here locally for some works. Not quite sure what conditions trigger though. Possibly a mix of child works with files & no files? Parent works having both files and children works? I'll try to track that down.

File: config/locales/rdr.en.yml [hunk 3]
```diff
@@ -18,6 +18,7 @@ en:
     unexpected_multiple_results: "Unexpected multiple results for %{identifier}"
     user_not_found: "User %{user_key} could not be found"
     not_found: "%{target} could not be found"
+    work_files_stats: "%{count} files (%{size})"
```
Comment (mikedaul) [inline config/locales/rdr.en.yml]: @coblej - could we change this to use the 'one/other' options in order to account for single files?
(see https://robots.thoughtbot.com/pluralizing-rails-i18n-translations)

File: app/views/hyrax/datasets/_work_files.html.erb (hunk 1 above)
Comment (seanaery) [reply app/views/hyrax/datasets/_work_files.html.erb]: @coblej I agree with @mikedaul on the presentation: might benefit from a minor tweak here. I think it'd look better if it didn't mimic the style used for the citation.

File: app/services/work_files_scanner.rb (hunk 2 above)
Comment (mikedaul) [reply app/services/work_files_scanner.rb]: I see the same error, but only for one item in my local instance. It's got three files in it - not sure why it's choking when other local items are not...

Comment (coblej) [reply app/services/work_files_scanner.rb]: I'll take a look and see if I can replicate.  Thanks.

File: config/locales/rdr.en.yml (hunk 3 above)
Comment (coblej) [reply config/locales/rdr.en.yml]: @mikedaul Thanks for the link.  Fixing this turned out to be much easier than I thought based on something else I had seen earlier.

**Comment by coblej** [comment]
@mikedaul , @seanaery Ready for your re-review.  I made two of the suggested changes -- (a) don't throw error if FileSet Solr document is missing file size field and (b) singular/plural in I18n translation.  I didn't change the `<dl>` treatment of the data on the `show` page, which Sean may work on restyling later.

**Pull request merged**